src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -569,4 +570,33 @@ public function setBodyFromAst(array $nodes)
         };
         $this->node->stmts = $validator(...$nodes);
     }
+
+    /**
+     * Add a new parameter to the method/function.
+     *
+     * @param string $parameterName
+     */
+    public function addParameter($parameterName)
+    {
+        $this->node->params[] = new ParamNode($parameterName);
+    }
+
+    /**
+     * Remove a parameter from the method/function.
+     *
+     * @param string $parameterName
+     * @return bool
+     */
+    public function removeParameter($parameterName)
+    {
+        $lowerName = strtolower($parameterName);
+
+        foreach ($this->node->params as $key => $paramNode) {
+            if (strtolower($paramNode->name) === $lowerName) {
+                unset($this->node->params[$key]);
+            }
+        }
+
+        return false;
```

Comment:
Why do you always return `false`? If at least one matching node was found, you should return `true`, no?